### ISSUE TITLE
Allow for lattice names in --isa (e.g. "Aspen-4-4Q-A").

### ISSUE THREAD

**Issue by notmgsk** [rigetti/quilc#310]
"8q-qvm" etc

**Comment by ecpeterson** [comment]
Seems reasonable to me!

**Comment by erichulburd** [comment]
@notmgsk Is this still relevant? Not seeing references to isa outside tests where they are just parsed with `yason`.

Are you looking to make use of the lattice names or just prevent them from breaking a parser?

**Comment by notmgsk** [comment]
@erichulburd Yeah that's my thought. Mimic the behaviour (where appropriate) of pyQuil's `get_qc`.

**Comment by notmgsk** [comment]
One issue is that the forest server endpoints require HTTPS, and so if we were to pull data from forest server (e.g. for "Aspen-*" lattices) then we would introduce an SSL dependency. That's not a terrible thing, but it would likely mean that we have to distribute an SSL library with the SDK. We should decide whether the cost of that extra dependency is worth the utility of this feature.
